subprojects/test-runner/test-robolectric-runner/build.gradle.kts

```diff
@@ -0,0 +1,10 @@
+plugins {
+    id("convention.kotlin-jvm")
+    id("convention.publish-kotlin-library")
+}
+
+dependencies {
+    api(libs.robolectric)
+    api(libs.junit)
+    implementation(projects.subprojects.logger.logger)
+}
```

Comment:
Do we need "test" prefix for modules inside "test-runner" directory?

"test-runner/robolectric" or "test-runner/robolectric-runner" looks enough. It already shows that these modules are test-runner parts.

Reply:
No, we don't. Added the prefix for consistency with existing modules. I'm going to remove them in one of the further PRs.